mora/lora.py

```diff
@@ -145,6 +140,66 @@
 }
 
 
+class AuthorizedSession(requests.Session):
+
+    def __init__(
+        self,
+        context=None,
+        retries=0,
+        max_retries=5,
+        import_settings=settings
+    ):
+
+        # Init Session
+        super().__init__()
+
+        # Set retry values
+        self.retries = retries
+        self.max_retries = max_retries
+
+        # Set user agent header
+        self.headers['User-Agent'] = 'MORA/0.1'
+
+        # SSL signature verification
+        self.verify = import_settings.CA_BUNDLE or True
+
+        # Fetch token from context
+        self.get_token_from_context(context)
+
+    def get_token_from_context(self, context=False):
+        """
+        Retrieve user token from application context
+        and set the authorization header for backend (lora) requests.
+
+        ("http://flask.pocoo.org/docs/0.12/api/?highlight=g#flask.g")
+
+        In "app.py":
+            attach_user_token_to_context()
+
+        is fetching the token from the authorization header
+        and attaching the token value to the application context.
+
+        :param context:     Application context
+                            For more information,
+                            head to the official flask documentation.
+        """
+
+        # Checks
+        if not hasattr(context, "user_token"):
+            raise AttributeError(
+                'Missing user_token attribute'
+            )
+
+        # TODO: Replace exception with proper exception type
+        if not context.user_token:
+            raise ValueError(
```

Comment:
The proper type would be `HTTPException` from `mora/exceptions.py`, but this is not the right place to raise it, as earlier mentioned.